lib/lrama/states.rb

```diff
@@ -524,5 +541,52 @@ def compute_default_reduction
         end.first
       end
     end
+
+    def split_states
+      transition_queue = []
+      @states.each do |state|
```

Comment:
I'm wondering it needs to limit the iteration only for the LALR states like `@states[0...@states.size].each ...`, in other words do we need to iterate states which are created by `#compute_state`?

Reply:
It might be my misunderstandings. We may need to iterate all states which includes newly created one.